redactrus_test.go

```diff
@@ -169,6 +169,14 @@ func TestStringer(t *testing.T) {
 
 	assert.Nil(t, err)
 	assert.Equal(t, logrus.Fields{"Stringer": "[REDACTED] is fmt.Stringer"}, logEntry.Data)
+
+	var s *stringerValue
+	nilStringerEntry := &logrus.Entry{
+		Data: logrus.Fields{"Stringer": s},
+	}
+	err = h.Fire(nilStringerEntry)
+
+	assert.Nil(t, err)
```

Comment:
```suggestion
	assert.Nil(t, err)
 
	var stringer fmt.Stringer
	nilStringerEntry = &logrus.Entry{
		Data: logrus.Fields{"Stringer": stringer},
	}
	err = h.Fire(nilStringerEntry)
	assert.Nil(t, err)
```

Reply:
Another go check for this. Passed locally for me. Requires also adding `fmt` to the import list at the top of the file.

Reply:
Thanks @ghthor!